src/api/admin/types.ts

```diff
@@ -706,6 +706,43 @@ export interface FullStateDump {
   integration_dump: FullIntegrationStateDump;
 }
 
+/**
+ * @public
+ */
+export interface DnaStorageInfo {
+  authored_data_size: number;
+  authored_data_size_on_disk: number;
+  dht_data_size: number;
+  dht_data_size_on_disk: number;
+  cache_data_size: number;
+  cache_data_size_on_disk: number;
+  used_by: Array<InstalledAppId>;
+}
+
+/**
+ * @public
+ */
+export interface DnaStorageBlob {
+  Dna: DnaStorageInfo;
```

Comment:
```suggestion
  dna: DnaStorageInfo;
```

Reply:
This has to be fixed upstream in Holochain. The Conductor API should produce all lower case snake case fields/properties.

Reply:
fixed by https://github.com/holochain/holochain/pull/2234